### ISSUE TITLE
API not returning title or thumbnail

### ISSUE THREAD

**Issue by joanarcas** [drawrowfly/amazon-scraper#131]
Hi,

I've been using this library for long time but today the API has start returning me the products array without some elements like thumnbnail or the title. Is anyone else experimenting the same issue?

**Comment by kchima** [comment]
Yup, just started using this API and I have noticed the same thing

**Comment by kchima** [comment]
Has there been any progress on this? The API results lacking title and thumbnail sort of makes this lib useless for most intents and purposes, so it is discouraging to see a total lack of activity on this and other issues / PRs. Is this project dead?

**Comment by Anuragd275** [comment]
Yup, I think this project is dead now.
Removing the title and thumbnail makes no sense.
